packages/@okta/vuepress-site/guides/sign-into-spa/create-okta-application/index.md

```diff
@@ -2,18 +2,18 @@
 title: Create an Okta Application
 ---
 
-Before your code can redirect to Okta to sign a user in, you need to create an Okta Application that represents your single-page application.
+Before you can sign a user in, you need to create an Okta Application that represents your single-page application.
 
-First, sign in to the Okta Developer Console:
+Start by signing in to the Okta Developer Console:
 
 <a href="https://login.okta.com/" target="_blank" class="Button--blue">Go to Console</a>
 
-Select **Applications**, then **Add Application**. Pick **Single-Page App (SPA)** as the platform. Enter a name for your application (or leave the default value).
+1. Select **Applications**, then **Add Application**. Pick **Single-Page App (SPA)** as the platform. Enter a name for your application (or leave the default value).
 
-Add the **Base URI** of your application that you use for local development, such as `http://localhost:8080`. Also, add any base URIs where your application runs in production, such as `https://app.example.com`.
+2. Add the **Base URI** of your application during local development, such as `http://localhost:8080`. Also, add any base URIs where your application runs in production, such as `https://app.example.com`.
 
-Next, enter values for the **Login redirect URI**. This is the callback route you created in the [previous step](/guides/sign-into-spa/-/define-callback). You should add values for local development (such as `http://localhost:8080/implicit/callback`) and production (such as `https://app.example.com/implicit/callback`).
+3. Next, enter values for the **Login redirect URI**. This is the callback explained in the [previous step](../-/define-callback-route/). Add values for local development (such as `http://localhost:8080/authorization-code/callback`) and production (such as `https://app.example.com/authorization-code/callback`).
```

Comment:
`../-/define-callback-route/` should be `../define-callback-route/`

Reply:
This syntax wasn't working for me, does it work for you?